### PR TITLE
reduce padding size

### PR DESCRIPTION
**Pull request by pnadolny13**
The spacing looks too big between sections. I don't know HTML/CSS very well so there might be a better way to do this but when I reduced the size below pt-20 it resolved. The weird thing is that the adjustments from pt-1 to pt-19 didnt really  change much but then at pt-20 it jumps and the gap is too big so I couldnt get anything in between.

Before:
Screen Shot 2023-03-31 at 12 20 36 PM

After:
Screen Shot 2023-03-31 at 12 20 43 PM

### CODE REVIEW

**Comment by tayloramurphy** [comment]
I'd like either @javajunky or @cjohnhanson to take a look. I'm not a huge fan of the adjustment b/c the headers are just too close to the text above them.

**Comment by pnadolny13** [comment]
@tayloramurphy same. I would love something in between the two but I couldnt get it to work.

**Comment by pnadolny13** [comment]
@javajunky @cjohnhanson friendly reminder 😄 . Do either of you know how to implement this? It seems like it should be a simple change.

**Comment by javajunky** [comment]
@pnadolny13 I think `pt-8` looks good. `pt-20` is 80px. `pt-8` is 32px. `pt-15` isn't a supported value which is why it is so close to the previous line.

This is `pt-8`:
Screenshot 2023-05-16 at 12 08 09 PM

File: src/components/PluginCapabilitiesSection.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <p class="text-3xl pb-4 pt-20 font-bold font-hg" id="capabilities">Capabilities</p>
+    <p class="text-3xl pb-4 pt-15 font-bold font-hg" id="capabilities">Capabilities</p>
```
Comment (pnadolny13) [inline src/components/PluginCapabilitiesSection.vue:3]:
```suggestion
    <p class="text-3xl pb-4 pt-8 font-bold font-hg" id="capabilities">Capabilities</p>
```

File: src/components/PluginCommandsSection.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div v-if="commands?.length">
-    <p class="text-3xl pb-4 pt-20 font-bold" id="commands">Commands</p>
+    <p class="text-3xl pb-4 pt-15 font-bold" id="commands">Commands</p>
```
Comment (pnadolny13) [inline src/components/PluginCommandsSection.vue:3]:
```suggestion
    <p class="text-3xl pb-4 pt-8 font-bold" id="commands">Commands</p>
```

File: src/components/PluginHelpSection.vue [hunk 3]
```diff
@@ -1,13 +1,13 @@
 <template>
   <div class="help-module">
-    <p class="text-3xl pb-4 pt-20 font-bold font-hg" id="contribute">Something missing?</p>
+    <p class="text-3xl pb-4 pt-15 font-bold font-hg" id="contribute">Something missing?</p>
```
Comment (pnadolny13) [inline src/components/PluginHelpSection.vue:3]:
```suggestion
    <p class="text-3xl pb-4 pt-8 font-bold font-hg" id="contribute">Something missing?</p>
```

File: src/components/PluginHelpSection.vue [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
     <p>This page is generated from a YAML file that you can contribute changes to.</p>
     <a
       v-if="name"
       :href="`https://github.com/meltano/hub/blob/main/_data/meltano/${plugin_type_plural}/${name}/${variant}.yml`"
       >Edit it on GitHub!</a
     >
-    <p class="text-3xl pb-4 pt-20 font-bold font-hg" id="looking-for-help">Looking for help?</p>
+    <p class="text-3xl pb-4 pt-15 font-bold font-hg" id="looking-for-help">Looking for help?</p>
```
Comment (pnadolny13) [inline src/components/PluginHelpSection.vue:10]:
```suggestion
    <p class="text-3xl pb-4 pt-8 font-bold font-hg" id="looking-for-help">Looking for help?</p>
```

File: src/components/PluginSettingsSection.vue [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <p class="text-3xl pb-4 pt-20 font-bold" id="settings">Settings</p>
+    <p class="text-3xl pb-4 pt-15 font-bold" id="settings">Settings</p>
```
Comment (pnadolny13) [inline src/components/PluginSettingsSection.vue:3]:
```suggestion
    <p class="text-3xl pb-4 pt-8 font-bold" id="settings">Settings</p>
```

File: src/templates/Plugins.vue [hunk 6]
```diff
@@ -181,7 +181,7 @@
                     </ul>
                   </span>
                   <div>
-                    <p class="text-3xl pb-4 pt-20 font-bold font-hg" id="getting-started">
+                    <p class="text-3xl pb-4 pt-15 font-bold font-hg" id="getting-started">
```
Comment (pnadolny13) [inline src/templates/Plugins.vue:184]:
```suggestion
                    <p class="text-3xl pb-4 pt-8 font-bold font-hg" id="getting-started">
```

**Comment by pnadolny13** [comment]
@javajunky thanks for the suggestion! That looks great, I updated the PR with the suggested change.

**Pull request merged**